TreeMaker/python/TMEras.py

```diff
@@ -10,6 +10,9 @@ def __init__(self):
             'TM2016_80X',
             'TM2017',
             'TM2018',
+            'TMUL2016',
```

Comment:
these won't actually work until we upgrade TreeMaker to be compatible with 10_6_X, right?

Reply:
Well, there's no scenario that references them right now, so it's not something I've actually tested. You're right that UL is 10_6_X and that TreeMaker probably needs to be updated to work with that. Still, there's no harm in having these eras available so that we can slowly add these sorts of features.